consumer/core/src/main/java/org/arquillian/algeron/consumer/core/ContractsPublisher.java

```diff
@@ -42,7 +42,7 @@ public void publish(@Observes AfterClass event, AlgeronConsumerConfiguration alg
             }
         }
 
-        throw new IllegalArgumentException(String.format("No pact publisher registered with name %s.", name));
+        throw new IllegalArgumentException(String.format("No contract publisher registered with name %s.", name));
```

Comment:
Well for me it is an argument is wrong (provider name is not registered) I think it could be State or Argument both valid depending on how you see.